<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyfieldsDialog</class>
 <widget class="QDialog" name="CopyfieldsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy values</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-copy">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>253</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="fieldsLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="nullCheck">
     <property name="text">
      <string>Update fields to empty values</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CopyfieldsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CopyfieldsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>CopyfieldsDialog</receiver>
   <slot>newButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newButtonClicked()</slot>
 </slots>
</ui>
